iob_soc.py

```diff
@@ -302,19 +301,23 @@ def __init__(self, *args, **kwargs):
                 "USE_MUL_DIV": USE_MUL_DIV,
                 "USE_EXTMEM": USE_EXTMEM
             },
-            io_connections={
-                "clk_i": clk,
-                "arst_i": cpu_reset,
-                "cke_i": cke,
-                "boot_i": boot,
-                "trap_o": trap,
+            connect={
+                "clk_i": get_wire_from_group("clk_en_rst", "clk"),
```

Comment:
No explocit suffix.  _I and _o managed automatically.

Reply:
Removed explicit port suffixes in commit [b3765b7](https://github.com/IObundle/iob-soc/pull/813/commits/b3765b71e2cfe4540777af2e32e0ee167e2763a9)